Sources/RichTextKit/Component/RichTextViewComponent+Alignment.swift

```diff
@@ -33,7 +33,7 @@ public extension RichTextViewComponent {
         _ alignment: RichTextAlignment
     ) {
         if richTextAlignment == alignment { return }
-        if !hasTrimmedText {
+        if !hasTrimmedText || !hasSelectedRange {
```

Comment:
We should make this function call setAttributes which decides internally if has selected range or not. But that is music of the future.